README.md

```diff
@@ -0,0 +1,111 @@
+<p align="center">
+	<img src="docs/eoslocal.png" width="600">
+</p>
+
+<p align="center">
+	<a href="#">
+		<img src="https://img.shields.io/dub/l/vibe-d.svg" alt="MIT">
+	</a>
+</p>
+
+# EOS Mainnet Local Network
+
+## Description
+EOS Local provides a quick way to setup an EOS local network for development.
+
+The primary benefits of containers are consistency across different environments and deployment ease.
+
+This project has a [full-stack-boilerplate](https://github.com/eoscostarica/full-stack-boilerplate) for your dApp. 
```

Comment:
This project works along with a [full-stack-boilerplate](https://github.com/eoscostarica/full-stack-boilerplate) to help you build your EOS dApp.